android/src/main/java/com/twiliovoicereactnative/ReactNativeArgumentsSerializer.java

```diff
@@ -251,7 +251,7 @@ public static WritableMap serializeCallMessage(@NonNull final CallMessage callMe
       new Pair<>(VoiceEventSid, callMessage.getVoiceEventSID()),
       new Pair<>(CallMessageContent, callMessage.getContent()),
       new Pair<>(CallMessageContentType, callMessage.getMessageContentType()),
-      new Pair<>(CallMessageMessageType, callMessage.getMessageType())
+      new Pair<>(CallMessageMessageType, String.valueOf(callMessage.getMessageType()))
```

Comment:
do it like this...
`new Pair<>(CallMessageMessageType, callMessage.getMessageType().name()))`